exercises/conor_finn/ex01_basics/src/main.h

```diff
@@ -0,0 +1,26 @@
+#ifndef MAIN_H_
+#define MAIN_H_
+
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+bool isBadPunct(char c);
+
+int wordCounterInterface();
+
+int main(int, char **);
+
+void cleanWord(std::string &s);
+
+void toLowerCase(std::string &s);
+
+std::vector<std::pair<int, std::string>> sortMap(std::map<std::string, int>);
```

Comment:
[Always specify the variable name as well as the type in function prototypes](http://developer.mantidproject.org/Standards/CPPStandards.html#functions-and-variables). With any luck the variable will have a descriptive name and will help self-document the function.